packages/mobile/e2e/README.md

```diff
@@ -157,15 +196,16 @@ class ExampleInput extends React.Component {
 
 It is recommended to follow the scheme parentID/ChildDescription.
 
-## Mocks for the e2e tests
+<!-- ## Mocks for the e2e tests
```

Comment:
Why is this section commented? Should we uncomment or delete it?

Reply:
I haven't tried this method of mocking; however, [Detox maintainers have said they are releasing a new mocking guide soon](https://github.com/wix/Detox/pull/3097) so I've commented it out as a reminder to update once the new guide is released. I'd like to covert mocking more in depth once the new Detox mocking guide is released.

Reply:
removed in https://github.com/valora-inc/wallet/pull/2277/commits/a3d63258fe3e1c9da01137d04a0290c6d4c7c60f!